docs/README-K8S.md

```diff
@@ -360,3 +360,9 @@ For example: Assume your network name is "ABCNET"  in network configuration, Set
 "Cloud API Owned" =  "True"
 "Tenant ID" =  "Testing"
 ```
+
+Troubleshoot
+------------
+
+If you get a message ``` Cloud Network Automation License not available or User not having sufficient permissions.``` in the cni-infoblox-daemon log then you have to check for the
```

Comment:
(see my comment above)
So, here's where you explain the user permission.

However, I don't think it belongs to README-K8S.md, since this is NOT k8s deployment specific. At least, this shouldn't ONLY be in README-K8s.md.

Reply:
Moved to common README.md